.gitignore

```diff
@@ -113,3 +113,6 @@ example/fast-neural-style/fast-neural-style/
 
 # CoreML models
 *.mlmodel
+
+virtualenv
```

Comment:
nit: people normally don't create a virtualenv inside the source checkout

Reply:
I usually do. Since this is just a .gitignore entry, it doesn't force anyone to do this, but enables it for people who want to. I'll keep it here.
On a side note: Is there a good reason to put virtualenv outside of the source directory I'm not aware of?